### PR TITLE
pr132

### PR DESCRIPTION
**Pull request by zzzeek**
change some things.

### CODE REVIEW

**Comment by sqla-tester** [comment]
New Gerrit review created for change 34e6793196eca4e5cd3e457aa870b16352acf3cc: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/testgerrit/+/1044

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Patch Set 1:

OK here's a plain comment.

File: bar.py [hunk 1]
```diff
@@ -13,14 +13,14 @@ def bar_one(x, y):
...
-        bat.do_q(q)
-    else:
-        bat.do_q(q + 7)
+    bat.do_q(q + name)
 
 
 def bar_three(a, g, h):
     bat.do_a(a)
     bat.do_g(g)
-    bat.do_h(h)
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

this was removed.

File: bar.py [hunk 2]
```diff
@@ -13,14 +13,14 @@ def bar_one(x, y):
 
 def bar_two(z, q):
     bat.do_z(z)
-    if name:
-        bat.do_q(q)
-    else:
-        bat.do_q(q + 7)
+    bat.do_q(q + name)
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

review comment.   code example:

    from foo import bar
    print(x + y)

like that.

File: bar.py (hunk 1 above)
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

this was removed.

File: bar.py (hunk 2 above)
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

review comment.   code example:

    from foo import bar
    print(x + y)

like that.

File: bar.py (hunk 1 above)
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

OK!

File: bar.py (hunk 2 above)
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

here's what a reply to that looks like.

Comment (zzzeek) [reply bar.py]: this is a github only reply

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Patch Set 1:

(2 comments)

File: bar.py (hunk 1 above)
Comment (sqla-tester) [reply bar.py]: **mike bayer** (zzzeek) wrote:

OK!

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Patch Set 1:

(2 comments)

File: bar.py (hunk 2 above)
Comment (sqla-tester) [reply bar.py]: **mike bayer** (zzzeek) wrote:

here's what a reply to that looks like.

**Comment by sqla-tester** [comment]
**"mike bayer <mike_mp@zzzcomputing.com>"** (zzzeek) wrote:

lets do the next one

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1044 has been **abandoned**.  That means that at least for the moment I need to close this pull request. Sorry it didn't work out :(

**Pull request closed without merging**